src/network/peloton_server.cpp

```diff
@@ -125,7 +125,7 @@ void PelotonServer::SSLInit() {
   //TODO(Yuchen): change this.
   //load trusted CA certificates (peer authentication)
   if (SSL_CTX_load_verify_locations(ssl_context, certificate_file_.c_str(), nullptr) != 1) {
-    LOG_ERROR("Exception when loading root_crt!");
+    LOG_WARN("Exception when loading root_crt!");
```

Comment:
@ChTimTsubasa Are these actually errors? Can we safely downgrade them to warnings?

Reply:
I consulted @haoxianghua and @dengyuchenkit today and this line can be downgraded to warning with further information. I would address the richer information as well as a fix for the log throwing here in an upcoming PR.